index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Flipkart</title>
+    <link rel="stylesheet" href="header.css">
+    <link rel="icon" href="flip.jpeg">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.7.2/css/all.min.css"
+        integrity="sha512-Evv84Mr4kqVGRNSgIGL/F/aIDqQb7xQ2vcrdIwxfjThSH8CSR7PBEakCr51Ck+w+/U6swU2Im1vVX0SVk9ABhg=="
+        crossorigin="anonymous" referrerpolicy="no-referrer">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css" rel="stylesheet">
+
+    <!-- Latest compiled JavaScript -->
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/js/bootstrap.bundle.min.js"></script>
+</head>
+
+<body>
+    <!-- navbar -->
+    <header class="container-fluid bg-primary">
+        <div class="container d-flex justify-content-start align-items-center gap-3 h-100 " tabindex="0">
+
```

Comment:
You can remove tabindex.

```suggestion
        <div class="container d-flex justify-content-start align-items-center gap-3 h-100 ">

```

Reply:
tab index is removed https://github.com/Sakshi-Simform/flipkart-ui/pull/2/commits/de0fce096d3d4138e2835142e9019b931945ce87